tests/headingscommand.js

```diff
@@ -53,6 +53,15 @@ describe( 'HeadingsCommand', () => {
 				expect( command.value ).to.equal( format );
 			} );
 		}
+
+		it( 'should be equal to defaultFormat if format has not been found', () => {
```

Comment:
I am sorry, this is just [the way I are](https://www.youtube.com/watch?v=U5rLz5AZBIA).